### PR TITLE
Get rid of bugfix textlint rule and update .NET and eBPF

### PR DESCRIPTION
**Pull request by cartermp**
Not a fan of that bugfix rule, heh

### CODE REVIEW

File: .textlintrc.json [hunk 1]
```diff
@@ -83,10 +83,12 @@
...
         "Zend",
         "Zipkin",
+        [".net", ".NET"],
         ["3rd[- ]party", "third-party"],
-        ["bug[- ]fix(es)?", "bugfix$1"],
         ["cpp", "C++"],
-        ["dotnet", ".NET"],
+        ["[Dd]ot[Nn]et", ".NET"],
+        ["EBPF", ".eBPF"],
+        ["eb[Pp][Ff]", ".eBPF"],
```
Comment (chalin) [inline .textlintrc.json:91]: I suspect that the `.` was a copy-paste error. Also, maybe we can combine the two rules:

```suggestion
        ["[eE][bB][Pp][Ff]", "eBPF"],
```

Comment (svrnm) [inline .textlintrc.json:91]: As far as I understand the "documentation" for textlint-rule-terminology those regular expressions are case insensitive: https://github.com/sapegin/textlint-rule-terminology (see "        // RegExp (case-insensitive) → replacement" in the sample config

You should get the same result by just putting "eBPF" in the list.

File: .textlintrc.json [hunk 2]
```diff
@@ -83,10 +83,12 @@
         "YouTube",
         "Zend",
         "Zipkin",
+        [".net", ".NET"],
         ["3rd[- ]party", "third-party"],
-        ["bug[- ]fix(es)?", "bugfix$1"],
```
Comment (svrnm) [inline .textlintrc.json:87]: Do you want to add the reverse rule then?

File: .textlintrc.json (hunk 1 above)
Comment (chalin) [reply .textlintrc.json:91]: Oh, that's even better.

Comment (chalin) [reply .textlintrc.json:91]: I've adapted the suggestion above based on @svrnm's comment about the regex being case insensitive.

File: .textlintrc.json (hunk 2 above)
Comment (chalin) [reply .textlintrc.json:87]: Yeah, we should.

Comment (cartermp) [reply .textlintrc.json:87]: Eh, I don't know if we should add the reverse rule. Both are fine. I don't feel strongly about this though.

File: .textlintrc.json [hunk 3]
```diff
@@ -83,10 +83,11 @@
...
         ["cpp", "C++"],
-        ["dotnet", ".NET"],
+        ["[Dd]ot[Nn]et", ".NET"],
```
Comment (chalin) [inline .textlintrc.json:89]:
```suggestion
        ["dotnet", ".NET"],
```

File: .textlintrc.json (hunk 2 above)
Comment (chalin) [reply .textlintrc.json:87]: I think that we should opt for consistency, but we can add the reverse rule in a followup PR.

**Review by chalin — APPROVED** [review]
Other than the fix suggested inline for "dotnet", LGTM

**Comment by chalin** [comment]
Actually even the "dotnet" fix can go in the followup PR.

File: .textlintrc.json [hunk 4]
```diff
@@ -83,10 +83,11 @@
         "YouTube",
         "Zend",
         "Zipkin",
+        [".net", ".NET"],
```
Comment (chalin) [inline .textlintrc.json:86]: Wait, this rule is wrong(it should use `\\.`) but in any case, it should be merged with the original "dotnet" rule.

**Review by chalin — CHANGES_REQUESTED** [review]

File: .textlintrc.json [hunk 5]
```diff
@@ -83,10 +83,11 @@
...
         ["3rd[- ]party", "third-party"],
-        ["bug[- ]fix(es)?", "bugfix$1"],
         ["cpp", "C++"],
         ["dotnet", ".NET"],
```
Comment (chalin) [inline .textlintrc.json:89]:
```suggestion
        ["(\\.|dot)net", ".NET"],
```

**Review by chalin — APPROVED** [review]
Ok, let's get this in.

**Pull request merged**